org.eclipse.buildship.ui/plugin.xml

```diff
@@ -176,6 +182,11 @@
                 name="In Gradle Tasks View"
                 parentId="org.eclipse.ui.contexts.window">
         </context>
+        <context
+              id="org.eclipse.buildship.ui.context.gradleNatureContext"
+              name="Gradle Nature Context"
```

Comment:
We need a better naming here because this value is visible here: https://dl.dropboxusercontent.com/u/1159431/Screen%20Shot%202015-07-03%20at%2015.22.35.png